test/test_entity-manager.js

```diff
@@ -106,6 +106,66 @@ define(function (require) {
                 expect(manager.entityHasComponent(entity, 'Position')).to.be.true;
                 expect(manager.entityHasComponent(entity, 'Unit')).to.be.true;
             });
+
+            it('create event is raised from the manager to the processor', function () {
```

Comment:
Those should read like a sentence, starting with "it". It this case, it could be `it('sends the create event from manager to processors', function () {`. The same goes for the next test.